packages/contracts/src/plugins/governance/majority-voting/token/TokenVotingSetup.sol

```diff
@@ -235,37 +216,13 @@ contract TokenVotingSetup is PluginSetup {
             PermissionLib.NO_CONDITION,
             DAO(payable(_dao)).EXECUTE_PERMISSION_ID()
         );
-
-        // Revocation of permission is necessary only if the deployed token is GovernanceERC20,
-        // as GovernanceWrapped does not possess this permission. Only return the following
-        // if it's type of GovernanceERC20, otherwise revoking this permission wouldn't have any effect.
-        if (isGovernanceERC20) {
-            permissions[3] = PermissionLib.MultiTargetPermission(
-                PermissionLib.Operation.Revoke,
-                token,
-                _dao,
-                PermissionLib.NO_CONDITION,
-                GovernanceERC20(token).MINT_PERMISSION_ID()
```

Comment:
Removing the mint permission would technically be a breaking change. It can still be granted later, though

Reply:
Why would it be breaking ? we only remove mint permission inside uninstallation of plugin.

Reply:
I see. The point being that it would be "breaking" in the sense of version 1 having one behaviour and version 2 having a different behaviour. 

Not just adding a feature or fixing a bug, but instead changing the specs of what happens